Wording Changes for Calendar Baseline per Issue 135 (#136)
cisagov/ScubaGoggles: baselines/Google Calendar Minimum Viable Secure Configuration Baseline v0.1.md

### Policies

#### GWS.CALENDAR.1.1v0.1

> External Sharing Options for Primary Calendars SHALL be configured to "Only free/busy information (hide event details)".

Reviewer: ```suggestion
External Sharing Options for Primary Calendars SHALL be configured to "Only free/busy information (hide event details)."
```

### Policies

#### GWS.CALENDAR.3.1v0.1

> External sharing options for secondary calendars SHALL be configured to "Only free/busy information (hide event details)".

Reviewer: ```suggestion
External sharing options for secondary calendars SHALL be configured to "Only free/busy information (hide event details)."
```

### Policies

#### GWS.CALENDAR.4.1v0.1
Calendar Interop SHOULD be disabled.

- Rationale

>   - Prevent information exchange between Microsoft and Google calendars for users using both services, unless it is required by the organization.

Reviewer: ```suggestion
  - Minimize attack surface by not enabling this feature which relies on Exchange Web Services for information exchange between Microsoft and Google calendars, unless required by the organization.
```
